content/Consumer Privacy Regulations CCPA and GDPR - Pablo Blanco.md

```diff
@@ -0,0 +1,240 @@
+documenting IT procedures, performing risk assessments under certain conditions,
+notifying the consumer and authorities when there is a breach, and strengthening
+rules for data minimization.
+
+The GDPR protects similar private data as the CCPA and recommends that companies
+provide a “reasonable” level of protection for personal information. It also
+mandates that the business explains to the customer how their information will
+be used and to explicitly ask for their permission to collect and process it.
+
+No matter how influenced the CCPA may have been by the GDPR, there are some
+clear differences we should notice in each legislation. Although the two laws
+are quite similar, the CCPA differs from the GDPR in a few important ways:
+
+- **Business impacted**: whereas businesses of any size must comply with the
+  GDPR, the CCPA only impacts businesses that reach a certain size that
+  processes a certain amount of data from consumers.
+- **Penalties**: GDPR penalties are capped based on a company’s annual revenue,
+  on the other hand, CCPA penalties have no limit and are assessed per violation
+  and number of affected consumers.
+- **Opt-in/Opt-out**: under GDPR, businesses must have opt-in from consumers
+  before collecting data, in the case of CCPA, consumers must opt-out of data
+  collection.
+- **Third-party data sales**: for GDPR businesses must have consent from
+  customers before any third-party processing or sales its data, while the CCPA
+  requires businesses to simply notify the customer of a data sale or transfer
+  and allow them to refuse.
+
+In short, both regulations give more power to consumers and enable them to take
+action if they want their data to remain private, besides that, businesses are
+required to attend their requests.
+
+## Does CCPA apply to my business?
+
+Contrary to what some believe, CCPA doesn’t just apply to those businesses based
+within California, rather than that any business with customers in California
+can be affected if that business meets the criteria mentioned above.
+
+Under the CCPA, businesses must be aware of what is considered private data,
+locate and treat customers information in a secure way, even addressing their
+vendors to ensure that they are compliant too.
+
+To protect customer’s personal information, businesses need to know what data
+they have, where it resides, and how it is processed. In this way, they can set
+up the appropriate security measures to be compliant with this regulation.
+
+Companies must realize that the impact of the CCPA is no trivial matter. They
+have to begin preparing now to become compliant, if they don’t, they will face
+reputation damage, fines, and loss of customer’s trust. Being prepared for this
+kind of regulation will bring a smooth path for the futures ones will come and
+will open new business opportunities and process improvements, in other words,
+It will pay to be compliant, and consumers will value you for it.
```

Comment:
```suggestion
will open new business opportunities and process improvements, in other words,
it will pay to be compliant, and consumers will value you for it.
```